hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationBackupHandler.java

```diff
@@ -253,7 +254,9 @@ private Operation getBackupOperation(BackupAwareOperation backupAwareOp) {
         // set service name of backup operation.
         // if getServiceName() method is overridden to return the same name
         // then this will have no effect.
-        backupOp.setServiceName(op.getServiceName());
+        if (!(op instanceof EntryOffloadableSetUnlockOperation)) {
```

Comment:
Is this needed because `EntryOffloadableSetUnlockOperation` extends `UnlockOperation `?